openr/py/openr/cli/utils/utils.py

```diff
@@ -9,7 +9,6 @@
 
 
 import copy
-import curses
```

Comment:
We should file a task to cleanup this going to open source land at all. I feel we can use click functions to find out about color or just use `click.secho` everywhere and it will do the right thing

Reply:
We do but I just `return False` in that function